tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -329,89 +334,79 @@ def testee(a: Field[[IDim], float64]) -> Field[[IDim], np.float32]:
     )
 
 
-def test_offset_field(fieldview_backend):
-    a_I_arr = np.random.randn(size, size).astype("float64")
-    a_I_float = np_as_located_field(IDim, KDim)(a_I_arr)
-    a_I_float_1 = np_as_located_field(IDim, KDim)(
+def test_offset_field(fieldview_backend, cartesian_case):
```

Comment:
Is there a reason for not using `cases.allocate` and `cases.verify` here?

- If there is a good reason (please specify) for `a` to contain random values when the default is that each item is unique?

- Does `b` really have to depend on the contents of `a`? For what reason?

Also, `ffront_utils.size` should not be used anymore, instead use `cartesian_case.default_sizes`.

Reply:
the test is showing that `as_offset` with an offset_field of ones correctly outputs the same result as `Ioff[1]` and `Koff[1]` expect for the last entry so a and b need to have the same values. so I can make a unique but if I do: 

```python
b = cases.allocate(cartesian_case, offset_index_field_fo, "a").extend({IDim: (0, 1), KDim: (0, 1)})()
a = cases.allocate(cartesian_case, offset_index_field_fo, "a")()
```
does not work